### PR TITLE
Creation of Recent Tab

### PR DESCRIPTION
**Pull request by anshmittal01**
I along with my team have added a new feature called recent tab which displays the 10 most recently used jupyter notebooks. We have also included the delete from the list option for privacy concern. We removed fetch and pandas dependency previously used.

### CODE REVIEW

**Comment by prof-lupin** [comment]
Hey @gnestor @takluyver ! This is our solution to the Feature request issue #2653 . We have passed the checks and the Travis CI integration testing. Kindly look into it and tell us your reviews and regarding any further course of actions we should take.

**Comment by takluyver** [comment]
Thanks! Sorry no-one reviewed this sooner.

So, first thought: if you're proposing UI changes, it's a nice detail to provide a screenshot, so people can quickly see what it would be like without having to get the code and try it. Here's one for the recent tab:

Screenshot of Jupyter Notebook dashboard showing the recent tab

**Comment by anshumusaddi** [comment]
Thank you, @takluyver  sir, for the input we will keep this in mind in our future contributions. By the way are all the other details fine with this pull request?

File: notebook/templates/tree.html [hunk 1]
```diff
@@ -1,211 +1,252 @@
-{% extends "page.html" %}
```
Comment (takluyver) [inline notebook/templates/tree.html]: Github is showing that every line in this file has changed, which makes it hard to see what's really changed. I think you've probably changed the line ending style. Can you work out how to change it back to what it was before?

**Comment by prof-lupin** [comment]
@takluyver Thanks for reviewing the PR. We will try to remedy these issues as soon as possible as we are busy with other academic commitments right now. Would appreciate your further guidance in the near future.

**Comment by jtpio** [comment]
Thanks @anshmittal01 for your work on this :+1: 

Since the Notebook code base has completely changed since, and is now targeting Notebook 7 which is based on JupyterLab, such feature is unlikely to be implemented in the repo directly.

Also there is a `jupyterlab-recents` extension which provides a similar functionality: https://github.com/jupyterlab-contrib/jupyterlab-recents. It would still need to be updated to JupyterLab 4 and Notebook 7 (https://github.com/jupyterlab-contrib/jupyterlab-recents/issues/7) though.

**Pull request closed without merging**